CHANGELOG.md

```diff
@@ -25,6 +25,11 @@ All notable changes to this project will be documented in this file.
 - CKKS: some advanced methods related to automorphisms are now public to facilitate their external use.
 - CKKS: improved the consistency of the API for in-place and `[..]New` methods.
 - CKKS: added the method `NewCiphertextAtLevelFromPoly`, which creates a ciphertext at a specific level from two polynomials.
+- CKKS: updated precision stats struct, added L2 norm in the statistics and improved the command line prints. 
+- CKKS: improved the algorithmic complexity of `MultiplyByDiagMatrixBSGS` and updated the bootstrapping parameters accordingly.
+- CKKS: reduced overall code footprint and improved readability of algorithms in `linear_transform.go`.
```

Comment:
"code footprint" is a bit ambiguous. Do you mean there is a noticeable performance improvement for the user ? Otherwise I would just consider it as an internal change and leave this line out of the Changelog.